src/js/pages/organizer/Branding.jsx

```diff
@@ -57,8 +57,8 @@ export default class Branding extends React.Component {
     const data = {
       name: this.state.name,
       organization: this.state.organization,
-      starttime: moment(this.state.starttime).unix(),
-      endtime: moment(this.state.endtime).unix(),
+      starttime: moment(this.state.starttime,'YYYY-MM-DD[T]HH:mm').valueOf(),
```

Comment:
Small thing, but we might want to pull out the date format string in case it needs to change later.